hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -137,15 +137,15 @@
           name: "ENABLE",
           mubi: true,
           desc: '''
-                Setting this field to 0xA will enable the ENTROPY_SRC module.
+                Setting this field to kMuBiTrue will enable the ENTROPY_SRC module.
```

Comment:
do we want to include the width in the description (`kMuBi4True`)? `kMuBiTrue` will probably not exist...